### DISCUSSION TITLE
How to Add custom Summarizers for columns that are not directly present in your database table?

### DISCUSSION THREAD

**Discussion by saumanarsad** [filamentphp/filament#13720] [Help] [answered]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

Assume that you have a table products, with columns items and price_per_item . So you create a table component and derive the third column 'total_price' as a product of items and price_per_product. But  how do you compute the sum of all total prices?

**Comment by saumanarsad** [comment, marked as answer]
This seems to be working for it 
Summary:make(Summarizer::make()
->using(fn($query)=>$query->items * $query->price_per_product)
)

**Reply by cawecoy** [reply]
Really? I get an error:

```
Undefined property: Illuminate\Database\Query\Builder::$items

```

**Comment by Murkrow02** [comment]
I'm having the same issue! Anyone figure out a fix yet?

**Reply by Saumanarshad** [reply]
Have you tried the above mentioned solution?
